openff/interchange/drivers/openmm.py

```diff
@@ -44,15 +44,21 @@ def get_openmm_energies(
     platform : str, default="Reference"
         The name of the platform (`openmm.Platform`) used by OpenMM in this calculation.
     detailed : bool, default=False
-        Attempt to report energies with more granularity. Not guaranteed to be compatible with all values
-        of other arguments. Useful for debugging.
+        Attempt to report energies with more granularity. Requires `combine_nonbonded_forces=False`.
+        Not guaranteed to be compatible with all values of other arguments. Useful for debugging.
```

Comment:
Might be helpful to mention with which values it is compatible. Eg perhaps, if this interpretation is correct, 

```suggestion
        Not guaranteed to be compatible with non-default values of other arguments. Useful for 
        debugging.
```